doc/BUILD.md

```diff
@@ -85,7 +85,7 @@ build  deps  examples  incremental  liana-cli  liana-cli.d  lianad  lianad.d  li
 ```
 
 To build the whole wallet including the GUI, you'll need to install its [build and runtime
-dependencies](https://github.com/wizardsardine/liana/tree/master/gui#dependencies) first. Then run
+dependencies](https://github.com/wizardsardine/liana/tree/master/doc/TRY.md#system-dependencies) first. Then run
```

Comment:
https://github.com/wizardsardine/liana/tree/master/gui#dependencies details both the runtime and build time dependencies of the GUI. If any are missing they should be added there.

https://github.com/wizardsardine/liana/tree/master/doc/TRY.md#system-dependencies lists the system dependencies necessary to follow the guide.

Reply:
done